src/libraries/System.Drawing.Common/src/System/Drawing/Printing/Margins.cs

```diff
@@ -235,14 +235,6 @@ public override bool Equals([NotNullWhen(true)] object? obj)
         /// <summary>
         /// Provides some interesting information for the Margins in String form.
         /// </summary>
-        public override string ToString()
-        {
-            return "[Margins"
-                + " Left=" + Left.ToString(CultureInfo.InvariantCulture)
-                + " Right=" + Right.ToString(CultureInfo.InvariantCulture)
-                + " Top=" + Top.ToString(CultureInfo.InvariantCulture)
-                + " Bottom=" + Bottom.ToString(CultureInfo.InvariantCulture)
-                + "]";
-        }
+        public override string ToString() => $"[Margins Left={Left} Right={Right} Top={Top} Bottom={Bottom}]";
```

Comment:
Isn't this a behavior change? Interpolated strings current culture, not invariant.

Reply:
No,  these are all non-negative Int32s... culture doesn't influence the default formatting of such values.